src/feature.py

```diff
@@ -0,0 +1,134 @@
+import sys
+# insert at 1, 0 is the script path (or '' in REPL)
+sys.path.insert(1, '/Users/valericita/Documents/GitHub/ChemLAB-PL/src/chem_lex')
```

Comment:
We cant have these types of paths in here. This path is only available at your computer so it wont work on any computer. We need to change this to another way